templates/includes/footer.html

```diff
@@ -1,106 +1,324 @@
+            >
+          </li>
+          <li>
+            <a
+              href="http://twitter.com/texastribune"
+              title="Twitter"
+              class="external"
+              ga-event-category="navigation"
+              ga-event-action="footer link click"
+              ga-event-label="twitter"
+              >{{ icon('twitter', align_base=True,
+              extra_classes='c-site-footer__icon') }}Twitter</a
+            >
+          </li>
+          <li>
+            <a
+              href="http://youtube.com/user/thetexastribune"
+              title="YouTube"
+              class="external"
+              ga-event-category="navigation"
+              ga-event-action="footer link click"
+              ga-event-label="youtube"
+              >{{ icon('youtube', align_base=True,
+              extra_classes='c-site-footer__icon') }}YouTube</a
+            >
+          </li>
+          <li>
+            <a
+              href="http://instagram.com/texas_tribune"
+              title="Instagram"
+              class="external"
+              ga-event-category="navigation"
+              ga-event-action="footer link click"
+              ga-event-label="instagram"
+              >{{ icon('instagram', align_base=True,
+              extra_classes='c-site-footer__icon') }}Instagram</a
+            >
+          </li>
+          <li>
+            <a
+              href="http://www.linkedin.com/company/texas-tribune"
+              title="LinkedIn"
+              class="external"
+              ga-event-category="navigation"
+              ga-event-action="footer link click"
+              ga-event-label="linkedin"
+              >{{ icon('linkedin', align_base=True,
+              extra_classes='c-site-footer__icon') }}LinkedIn</a
+            >
+          </li>
+          <li>
+            <a
+              href="https://www.reddit.com/user/texastribune"
+              title="Reddit"
+              class="external has-xxs-btm-marg l-display-ib"
+              ga-event-category="navigation"
+              ga-event-action="footer link click"
+              ga-event-label="reddit"
+              >{{ icon('reddit', align_base=True,
+              extra_classes='c-site-footer__icon') }}Reddit</a
+            >
+          </li>
+          <div
+            class="has-notch has-notch--thin has-bg-yellow has-b-btm-marg"
+            aria-hidden="true"
+          ></div>
+          <li class="has-tiny-btm-marg t-lh-s">
+            <a
+              href="https://www.facebook.com/groups/thisisyourtexas/"
+              title="This is Your Texas"
+              class="external"
+              ga-event-category="navigation"
+              ga-event-action="footer link click"
+              ga-event-label="this is your texas"
+              >{{ icon('your-texas', align_base=True,
+              extra_classes='c-site-footer__icon') }}Join our Facebook Group,
+              This Is Your Texas.</a
+            >
+          </li>
+        </ul>
```

Comment:
Do you tend to prefer to have each attribute on a new line? Is there a certain formatter you use to do this? We should all probably start using it so our git history isn't just us reformatting things. ⚔️

Reply:
I have no preference! 😊  I believe the formatting for these files wasn't consistent with our prettier styles (defined in [package.json](https://github.com/texastribune/donations/blob/copyright-year/package.json)), which requires each line to be no more 80 characters. When I started editing the files VSCode auto-formatted them.

Reply:
Let me know if that's all good and I'll merge. I ask just in case there's something about our code styling that I don't understand. I'm happy to switch it back to its original formatting if needed.

Reply:
Ah gotcha, ok all good then! Yeah I guess I don't have prettier running on save. All good to merge to me!